tests/ui-toml/excessive_nesting/default/clippy.toml

```diff
@@ -0,0 +1 @@
+excessive-nesting-threshold = 0
```

Comment:
It would be good if this were 1 higher than whatever the other file ends up as rather than `0` (effectively every other test in the repo checks that nothing lints on `0` already)

Then if most of the tests only trigger the lower number you can visually scan the file to check if all the indents are at the right number